app/src/main/AndroidManifest.xml

```diff
@@ -260,6 +260,9 @@
     <activity
       android:name=".app.devoptions.forcenetworktype.testing.ForceNetworkTypeTestActivity"
       android:theme="@style/OppiaThemeWithoutActionBar" />
+    <activity android:name=".app.policies.PoliciesActivity"
+      android:label="@string/privacy_policy_title"
```

Comment:
We should default to a string agnostic of the two policies (maybe something like: Policy Page) and then specify the correct label at runtime (see other activities that need to do this, like ExplorationActivity). Please make sure to test with a screenreader that the label is properly defined for both policies.

Reply:
Done.